11-payment-encoding.md

```diff
@@ -55,23 +59,25 @@ The following `multiplier` letters are defined:
 ## Requirements
 
 A writer:
-  - MUST encode `prefix` using the currency it requires for successful payment
-  - If it requires a specific minimum amount for successful payment:
-	- MUST include that `amount`
-	- MUST encode `amount` as a positive decimal integer with no leading 0s
-	- SHOULD use the shortest representation possible by using the largest
-	  multiplier or omitting the multiplier
+  - MUST encode `prefix` using the currency required for successful payment.
+  - if a specific minimum `amount` is required for successful payment:
+	  - MUST include that `amount`.
+	- MUST encode `amount` as a positive decimal integer with no leading 0s.
```

Comment:
Seems we are mixing tabs and spaces here, could we make this consistent while we're here?